README.md

```diff
@@ -51,9 +53,26 @@ Unit tests for individual utility functions in the WAF WASM extension are
 available in `test` directory. To run them, execute from the root of the
 repository:
 ```
-source ./build_test.sh
+source ./unit_test.sh
+```
+
+## Integration Tests
+
+Before conducting integration tests, make sure that the WASM binary file has been built and the `istio/proxy` docker image has been pulled according to instructions in previous sections. The tests can be run by executing the following script from the root of
+the repository:
+```
+source ./integration_test.sh
```

Comment:
source is usually to "load" something that is not a shell script (like a list of environment variable, that you find in your .bashrc/.bash_profile).

When you want to execute a shell script, 2 possibilities, invoke sh : `sh ./integration_test.sh`
Or make integration_test.sh executable `chmod +x integration_test.sh` before committing it and execute it directly `./integration_test.sh` (the script needs to get the shell reference as a first non-comment line :
`#!/bin/sh` )

Reply:
Updated in this [commit](https://github.com/dengyijia/WAF-wasm/pull/3/commits/4cf76c481abf093d634bc9e9d527768c117ce31d)